msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 angle                            # Angle (in radians) in the azimuth plane between the sensor and the detected point.
```

Comment:
if this is the azimuth angle why call it angle? The most complete would be the full `azimouth_angle` but it can be shortened non-ambiguously to `azimuth`? Whereas angle is not disambiguated.

Reply:
+1 for azimuth

Reply:
Another +1 for azimuth

Reply:
azimuth is fine by me